server/views/pages/activities/activitiesMigration.njk

```diff
@@ -18,6 +19,17 @@
     <h1 class="govuk-heading-l">Activities migration</h1>
 
     <div>
+      {% set migrationInstructionsHtml %}
+        <p>Start a migration by pressing the "Start new migration" button and entering a single prison ID. You can optionally enter a single CRS_ACTY_ID to migrate a single Course Activity instead of all Course Activities. This will migrate <strong>active</strong> Course Activities from NOMIS to DPS. By "active" we mean the activity has not ended and has at least one prisoner allocated to attend.</p>
+        <p>All activities migrated to DPS will be synchronised back to NOMIS as <strong>new</strong> activities under the new program service codes. We are using a 1 way sync from DPS -> NOMIS so the old NOMIS activities are now redundant.</p>
+        <p>When the activity migration is complete and has been checked, click on the "Migrate Allocations" link and migrate the allocated prisoners to DPS. This will migrate all allocated prisoners onto the new activities and also synchronise them back to NOMIS. Note that ended or waiting allocations will not be migrated.</p>
+        <p>When the allocation migration is complete and has been checked we need to end all of the old NOMIS activities as they have been replaced by the migrated activities in DPS. Press the "End NOMIS activities" button to end all of the old activities and also end any allocations for those activities.</p>
```

Comment:
Add some instructions to the page to explain the process is to migrate activities, then migrate allocations before finally ending NOMIS activities.